ivy/functional/backends/tensorflow/experimental/linear_algebra.py

```diff
@@ -189,3 +189,24 @@ def cond(
             tf.linalg.inv(x), ord=p, axis=[-2, -1]
         )
     return k
+
+
+def solve_triangular(
+    a: Union[tf.Tensor, tf.Variable],
+    b: Union[tf.Tensor, tf.Variable],
+    /,
+    *,
+    lower: bool = True,
+    transpose: bool = False,
+    conjugate: bool = False,
```

Comment:
You should either handle the `conjugate` argument in all backends or skip it completely.